### DISCUSSION TITLE
Idea: Keep the "Last minute song" selection

### DISCUSSION THREAD

**Discussion by ux-git** [sircharlo/meeting-media-manager#2197] [Ideas] [answered]
### Is your idea related to a problem?

The "Last minute song add/change" method of adding a song has recommended itself as a great and convenient way to add a public talk song for a weekend meeting. It's less complicated, more visual and with less busy UI comparing to normal "Manage media" page. 

### Describe your idea

The idea is simple – to keep the selected song until it's manually removed. Sometimes we select an opening song this way, but then go into settings, or do somethings else – and the song gets cleared from the list.

Perhaps it's possible to add this feature and add a toggle into Settings to switch this on and off, since I suppose for some congregation current behaviour is preferable if they're using it just for quick program change actions.

<img width="1168" alt="Screenshot 2023-10-15 at 12 08 10" src="https://github.com/sircharlo/meeting-media-manager/assets/80993061/977e2801-20dd-487c-ae3a-ef1a0bf053ff">

### Describe alternatives you've considered

Use the normal "Manage media" page. It works, but since it's more complicated and less visual way off adding a song, brothers often forget to do this in advance, which adds additional time to start the song.

**Comment by mtdvlpr** [comment, marked as answer]
Hi @ux-git, it the upcoming refactor, we plan to always show the song picker, since it doesn't take up that much space.

**Reply by ux-git** [reply]
Fantastic idea, I was also thinking about it. This is going to make song selection even more intuitive 👍
